packtools/sps/sps_maker.py

```diff
@@ -53,3 +53,23 @@ def make_package_from_uris(xml_uri, renditions_uris_and_names=[], zip_folder=Non
     package_metadata['temp-zipfile'] = _zip_files_from_uris_and_names(zip_filename, uris_and_names, zip_folder)
 
     return package_metadata
+
+
+def _get_xml_sps_from_uri(xml_uri):
+    if xml_uri == '':
+        raise exceptions.SPSXMLLinkError('XML URI is empty. Please, informe a link address.')
+
+    if 'http' in xml_uri:
+        content = reqs.requests_get_content(xml_uri)
```

Comment:
@rafaelpezzuto acho que vale a pena pegar o httpcodeerror (para saber se foi indisponibilidade ou not found ou ...) e explicitar no raise

Reply:
Concordo. Farei isso.